academy/exchange/cloud/server.py

```diff
@@ -0,0 +1,400 @@
+"""HTTP message exchange client and server.
+
+To start the exchange:
+```bash
+python -m academy.exchange.http --host localhost --port 1234
```

Comment:
These examples will need to be updated. Right now it would be 
```
python -m academy.exchange.cloud.server
```
But I think it'd be better to be `academy.exchange.cloud`. You can do this by moving
```python
if __name__ == '__main__':
    raise SystemExit(_main())
```
at the end of the file to a new file named `academy/exchange/cloud/__main__.py` file that has:
```python
from academy.exchange.cloud.server import _main

if __name__ == '__main__':
    raise SystemExit(_main())
```